packages/auth/src/AuthProvider/useReauthenticate.ts

```diff
@@ -25,6 +25,13 @@ export const useReauthenticate = <TUser>(
   const getToken = useToken(authImplementation)
 
   return useCallback(async () => {
+    if (authImplementation.loadWhileReauthenticating) {
```

Comment:
Can we please have a comment here on why we don't just always do this? At first glance it sounds like an obvious thing to do (switch to "loading" while the auth provider is reauthenticating that is)

Reply:
Done in https://github.com/redwoodjs/redwood/pull/7852/commits/f8af3f3620f37322406ef3bafb82546d4800d4aa